CONTRIBUTING.rst

```diff
@@ -856,6 +856,11 @@ Commit title tags
   beginning of a commit message. Multiple tags are separated by space. The
   following tags are defined:
 
+  - ``h`` the commit is a temporary hotfix. These commit should be reverted
```

Comment:
Append this to the list, not prepend.

Reply:
`Tags must appear in a title in the order they are defined above, as in [u r R 1/2]. This ensures that more consequential tags appear earlier.`

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#36commit-title-tags

Just to confirm the hotfix commit tag should be after `M/N` commit tag?